### ISSUE TITLE
Final Report Peer Review

### ISSUE THREAD

**Issue by Adrianne-Fu** [juy48/ORIE-5741#12]
The aim of this project is to use linear regression model to predict stock performance (closing price) within the S&P500 equity index in US market. Financial status and are taken into account in terms of features used for model fitting.

Three things I like about the project:
- The team considered global abnormal environment during the pandemic as well as its impact on stock market and excluded the time period in following analysis, which would enhance the ability of  generalization of the models.
- Different feature selection methods were adopted in the project, including calculating collinearity, automatic method of stepwise selection, LASSO, and random forest. One thing notable is that the team performed feature selection evaluation both before and after different normalization.
- The team comprehensively discussed possible future improvement. The finding of different monthly opening stock price having different impact on annual close stock price performance is especially valuable and promising in terms of making more precise and real time prediction on a month level.

Three things might be improved:
- The data set doesn't include economic characteristics that may contain some important information during the time period, examples are national GDP levels, industry reshuffling, or global trade policies.
- The team randomly split the dataset into training set and validation set. Since the random process is not controllable, the data might be split unevenly, namely, companies of a certain type might be included in only training or validation set.
-  Feature improvement was considered but no desired results were generated. It might be helpful to consider non-technical method of including more data points by adding more companies or technical method of dataset  augmentation.
